Add note regarding redeploying metal3-dev-env (#190)
metal3-io/metal3-io.github.io: try-it.md

$ make clean
```

> info "Note"

> > in case of redeploying metal³-dev-env with a different release version of

Reviewer: ```suggestion
> When redeploying metal³-dev-env with a different release version of
```

Author: Thanks, updated.

> > CAPM3 requires `FORCE_REPO_UPDATE` variable in `config_${user}.sh` set to

Reviewer: ```suggestion
> CAPM3, you must set the `FORCE_REPO_UPDATE` variable in `config_${user}.sh` set to
```

Author: Thanks, updated.
